frontend/src/locales/scripts/en.json5

```diff
@@ -467,10 +467,30 @@
       },
     },
   },
-  generatedTags: {
-    title: "About Generated Tags",
+  tags: {
+    title: "Understanding Tags in {openverse}",
+    /** generatedTags.intro.a-b are parts of a single section explaining how tags work in Openverse. */
     intro: {
-      content: "Generated tags are automatically assigned to media items based on the content of the media item itself. These tags are generated using machine learning models trained on a large dataset of images and audio tracks.",
+      a: "Each creative work in {openverse} may have tags, an optional set of keywords used to describe the work and make it easier for users to find relevant media for their searches.",
+      b: "These tags fall into two main categories: source tags and generated tags. Understanding the difference between them can enhance your search experience and improve the accuracy of your results.",
+    },
+    sourceTags: {
+      title: "Source Tags",
+      /** sourceTags.content.a-b are parts of a single section explaining how source tags work in Openverse. */
```

Comment:
@obulat heads up that these comments will not appear in the generated pot file, only comments on the string properties themselves do. I have a small patch I was working on that might fix this though, because it would be nice if this just worked!